projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -564,6 +565,33 @@ function wpcom_launchpad_is_keep_building_enabled() {
 	return apply_filters( 'is_launchpad_keep_building_enabled', false );
 }
 
+/**
+ * Filter task visibility for the Keep building task list.
+ *
+ * @param array $task The task array.
+ *
+ * @return array The filtered array of task IDs.
+ */
+function wpcom_launchpad_keep_building_visible_tasks( $task ) {
```

Comment:
I think we should rename this parameter to `$task_list`.
```suggestion
function wpcom_launchpad_keep_building_visible_tasks( $task_list ) {
```